app/views/layouts/application.html.haml

```diff
@@ -9,6 +9,8 @@
       %ul.dropdown-menu
         %li= link_to t('.profile'), my_profile_path
         %li= link_to t('.ordergroup'), my_ordergroup_path
+        - if BankAccount.any?
+          %li= link_to t('.reference_calculator'), home_reference_calculator_path
```

Comment:
I guess people would use bank accounts without the reference stuff you're using?
What about a configuration option for using the references to direct where the money is for? Or at the very least, if there is a `name_short` somewhere ... but that would need some caching perhaps, since this is called on every page view.

Reply:
This can also be done later, when someone requests it. Fine.